File: 백준/Bronze/26711. A＋B/A＋B.java
import java.io.*;
import java.math.BigInteger;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        BigInteger a = new BigInteger(br.readLine());
        BigInteger b = new BigInteger(br.readLine());

        System.out.println(a.add(b));
    }
}

File: 백준/Bronze/1000. A＋B/A＋B.kt
import java.io.*
import java.util.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val st = StringTokenizer(br.readLine())
    val num1 = Integer.parseInt(st.nextToken())
    val num2 = Integer.parseInt(st.nextToken())

    println(addTwoNumber(num1, num2))
}

fun addTwoNumber(num1 : Int, num2 : Int) : Int{
    return num1 + num2
}